Audacia.Mail.Test.API/Controllers/MailController.cs

```diff
@@ -8,11 +8,11 @@ namespace Audacia.Mail.Test.API.Controllers;
 public class MailController : Controller
 {
     private readonly IMailService _mailService;
-    private readonly IMailClientHandlerFactory _mailClientHandlerFactory;
+    private readonly IMailClientFactory _mailClientHandlerFactory;
```

Comment:
nit: can the field (and ctor parameter) names be changed as well?

Reply:
my bad it's fixed now